ci/rspec-audit/migratedb

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash -e
 # Run DB migration for audit database
 BUNDLE_GEMFILE=./Gemfile \
-  bundle exec sequel $AUDIT_DATABASE_URL \
+  bundle exec sequel $DATABASE_URL \
```

Comment:
can you please explain in the commit message why you did this change?

Reply:
added more details in the commit messsage